scripts/setup_devmode.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env sh
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2013, 2014, 2015 CERN.
+#
+# Invenio is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+#
+#
+# Sets the configuration of Invenio to defaults which are sutable for
+# developers.
+#
+# !!!                  WARNING                      !!!
+# !!! Do NOT use these configurations in production !!!
+#
+
+echo "Load developer configuration. Do NOT use this in production mode!"
+inveniomanage config create secret-key
+inveniomanage config set CFG_EMAIL_BACKEND flask_email.backends.console.Mail
```

Comment:
+1 for making these changes. While you improving it, it might be worthwhile reducing the number of calls to ``inveniomanage config`` since each call takes a couple of seconds and all they do is to write to a text file. Something along the lines of:

```
$ cat <<EOF >> var/invenio.base-instance/invenio.cfg
PACKAGES_EXCLUDE = []
...
EOF
```